tests/features/paragraphs/custom_page.paragraphs.feature

```diff
@@ -71,3 +71,46 @@ Feature:
     Then I should see the success message "Custom page Don't Mess with the Zohan has been updated."
     And I should see "I'm half Australian, half Mt. Everest"
     And the response should contain "{\"foo\":\"bar\"}"
+
+  @javascript
+  Scenario: Add an accordion to the custom page.
+    Given I am logged in as a moderator
```

Comment:
Wouldn't it be better to use the facilitator as the editor of the custom page? I don't think it is a common task for moderators to create content in collections.